### ISSUE TITLE
Host collective page: hosted collective/event filters

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#2971]
While discussing with admins of https://opencollective.com/paris recently, I got the feedback that it is more complex to see hosted collectives with the new collective page because they were bundled under the the same `Fiscal host` filter than collectives.

What they would like to have is a clear list of the collectives they are hosting. While the new host dashboard will certainly solve this, I believe there's a low hanging fruit for us to make things clearer on the new collective page for hosts:

**Instead of one `Fiscal Host` filter, we could have two specific filters:**
- `Hosted Collectives`
- `Hosted Events`

---

![image](https://user-images.githubusercontent.com/1556356/76215933-015b9780-6210-11ea-9412-23a1a94fde55.png)

**Comment by davejon1337** [comment]
can i try this out?

**Comment by Betree** [comment]
@davejon1337 sure, thanks for jumping in! @piamancini is the owner of the collective page project, let's just wait for her to confirm this before starting with the implementation.

**Comment by piamancini** [comment]
Thanks @Betree It's definitely an easy win. 
@davejon1337 You can go ahead!

**Comment by patcon** [comment]
Just a small note that I find that sub-collective's events showing prominently on host page to be a bit confusing 🙏 

Example: https://opencollective.com/hyphacoopinc#category-CONTRIBUTIONS

It used to be that the "host page" by default had a pretty clear list of our hosted collectives. This would strike me as the main interest of someone visiting our host page. But now, once a collective has created an event, it now would appear to add each event to our host page in a way that seems noisy and cluttered. Yes, it can be filtered, but the default list will seemingly become noisy if we choose to keep using events. (Seems like a disincentive for hosts to encourage using events.)

Am I understanding correctly my options here? :)

(Also, happy to create a new issue if this is "real"!)

**Comment by alanna** [comment]
@patcon I agree - I raised #4128
